secator/query/api.py

```
@@ -88,3 +88,51 @@ def _execute_update(self, query: dict, update: dict) -> int:
+            return []
+
+    def get_workspace(self, workspace_id: str):
+        """Get workspace info from API."""
+        try:
+            endpoint = CONFIG.addons.api.workspace_get_endpoint.format(workspace_id=workspace_id)
+            return self._make_request('GET', endpoint)
+        except Exception as e:
+            console.print(Warning(message=f'API get_workspace failed: {e}'))
+            return None
+
+    def list_runners(self, workspace_id: str = None, runner_type: str = None):
```

Ruff (0.15.15)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)